### DISCUSSION TITLE
LF2 MyOpenMath

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#370] [General]
Reviewer: @tdegeorge 

1-4 identify slope and point from equation
5 pick graph matching properties
6-14 find equations given different properties in different forms
15-18 find equation given graph
19-22 find equation from table

**Comment by tdegeorge** [comment]
Some of the questions are a little ambiguous. For example, problem 14 asks for the equation when given a slope and a point, but does not specify which form. The answer indicates that the answer should be in slope-intercept form, but how can we see if it will accept point-slope form? Or should we just change the directions so that it is clear which form we want?
<img width="474" alt="Screenshot 2024-07-04 at 10 14 50 AM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/143553140/6485cfce-a961-4dd8-aacb-2e2691fc94ba">

**Comment by tdegeorge** [comment]
I didn't see any questions that asked students to write the equation of a line in point-slope form specifically, although some of the answers show that they accept point-slope as well (I only saw problems 7 and 11 with solutions that also accepted point-slope).

**Comment by tdegeorge** [comment]
Although not specific in the objective (it states "determine the slope and y-intercept of a line given an equation"), the activity book does ask students in the last activity to find the slope and y-intercept when given standard form. Did we want to include 1-2 questions like that too?
